package.json

```diff
@@ -1,5 +1,8 @@
 {
 	"name": "preact-signals",
+	"version": "1.0.0",
+	"description": "Manage state with style in every framework",
+	"main": "index.js",
```

Comment:
```suggestion
```

Sorry, I removed these in #279 as they're rather useless for a monorepo root. Our mistake for still having them around!